shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -64,6 +64,8 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:@"SystemChrome.setPreferredOrientations"]) {
     [self setSystemChromePreferredOrientations:args];
     result(nil);
+  } else if ([method isEqualToString:@"SystemChrome.getPreferredOrientations"]) {
+    result([self getSystemChromePreferredOrientations]);
```

Comment:
This needs a test in https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm.  You should be able to do something like:
```
  FlutterEngine* engine = [[[FlutterEngine alloc] initWithName:@"test" project:nil] autorelease];
  [engine runWithEntrypoint:nil];
  FlutterViewController* flutterViewController =
      [[FlutterViewController alloc] initWithEngine:engine nibName:nil bundle:nil];

  id flutterViewControllerMock = OCMPartialMock(flutterViewController);
  OCMStub([flutterViewControllerMock supportedInterfaceOrientations]).andReturn(UIInterfaceOrientationMaskPortraitUpsideDown & UIInterfaceOrientationMaskLandscapeRight);
  engine.viewController = flutterViewControllerMock;

  std::unique_ptr<fml::WeakPtrFactory<FlutterEngine>> _weakFactory =
      std::make_unique<fml::WeakPtrFactory<FlutterEngine>>(engine);
  FlutterPlatformPlugin* plugin =
      [[[FlutterPlatformPlugin alloc] initWithEngine:_weakFactory->GetWeakPtr()] autorelease];
```
then call the `handleMethodCall` and check the result (there are some examples in that test file I think).